spec/ffi/function_spec.rb

```diff
@@ -64,14 +64,14 @@ class << self; self; end
 
   it 'can wrap a blocking function' do
```

Comment:
It is less broken than the current. The current implementation does test nothing. It succeeds with :blocking=>true and equally with :blocking=>false.

However this commit is not related to the appveyor result, and should be in a separate PR. I'll remove it here. Anyways it's probably better to test for concurrency, than for any timing...